reports-queue/reports-queue.yml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: reports-queue
```

Comment:
what about `report-service-queue`

Reply:
I think queue does not belong to report-service